internal/inspect/queries.go

```diff
@@ -0,0 +1,299 @@
+ORDER BY query_start`
+
+const LONG_RUNNING_QUERY = `SELECT
+pid,
+now() - pg_stat_activity.query_start AS duration,
+query AS query
+FROM
+pg_stat_activity
+WHERE
+pg_stat_activity.query <> ''::text
+AND state <> 'idle'
+AND now() - pg_stat_activity.query_start > interval '5 minutes'
+ORDER BY
+now() - pg_stat_activity.query_start DESC`
+
+const OUTLIERS_QUERY = `SELECT
+interval '1 millisecond' * total_exec_time AS total_exec_time,
+to_char((total_exec_time/sum(total_exec_time) OVER()) * 100, 'FM90D0') || '%'  AS prop_exec_time,
+to_char(calls, 'FM999G999G999G990') AS ncalls,
+interval '1 millisecond' * (blk_read_time + blk_write_time) AS sync_io_time,
+query
+FROM pg_stat_statements WHERE userid = (SELECT usesysid FROM pg_user WHERE usename = current_user LIMIT 1)
+ORDER BY total_exec_time DESC
+LIMIT 10`
+
+const REPLICATION_SLOTS_QUERY = `SELECT
+s.slot_name,
+s.active,
+COALESCE(r.state, 'N/A') as state,
+CASE WHEN r.client_addr IS NULL
+   THEN 'N/A'
+   ELSE r.client_addr::text
+END replication_client_address,
+GREATEST(0, ROUND((redo_lsn-restart_lsn)/1024/1024/1024, 2)) as replication_lag_gb
+FROM pg_control_checkpoint(), pg_replication_slots s
+LEFT JOIN pg_stat_replication r ON (r.pid = s.active_pid)`
+
+const ROLE_CONNECTIONS_QUERY = `SELECT
+rolname,
+(
+  SELECT
+    count(*)
+  FROM
+    pg_stat_activity
+  WHERE
+    pg_roles.rolname = pg_stat_activity.usename
+) AS active_connections,
+CASE WHEN rolconnlimit = -1 THEN current_setting('max_connections') :: int8
+     ELSE rolconnlimit
+END AS connection_limit
+FROM
+pg_roles
+ORDER BY 2 DESC`
+
+const SEQ_SCANS_QUERY = `SELECT relname AS name,
+seq_scan as count
+FROM
+pg_stat_user_tables
+ORDER BY seq_scan DESC`
+
+const TABLE_INDEX_SIZE_QUERY = `SELECT c.relname AS table,
+pg_size_pretty(pg_indexes_size(c.oid)) AS index_size
+FROM pg_class c
+LEFT JOIN pg_namespace n ON (n.oid = c.relnamespace)
+WHERE NOT n.nspname LIKE ANY($1)
+AND n.nspname !~ '^pg_toast'
+AND c.relkind='r'
+ORDER BY pg_indexes_size(c.oid) DESC`
+
+const TABLE_RECORD_COUNTS_QUERY = `SELECT
+relname AS name,
+n_live_tup AS estimated_count
+FROM
+pg_stat_user_tables
+ORDER BY
+n_live_tup DESC`
+
+const TABLE_SIZE_QUERY = `SELECT c.relname AS name,
+pg_size_pretty(pg_table_size(c.oid)) AS size
+FROM pg_class c
+LEFT JOIN pg_namespace n ON (n.oid = c.relnamespace)
+WHERE NOT n.nspname LIKE ANY($1)
+AND n.nspname !~ '^pg_toast'
+AND c.relkind='r'
+ORDER BY pg_table_size(c.oid) DESC`
+
+const TOTAL_INDEX_SIZE_QUERY = `SELECT pg_size_pretty(sum(c.relpages::bigint*8192)::bigint) AS size
+FROM pg_class c
+LEFT JOIN pg_namespace n ON (n.oid = c.relnamespace)
+WHERE NOT n.nspname LIKE ANY($1)
+AND n.nspname !~ '^pg_toast'
+AND c.relkind='i'`
+
+const TOTAL_TABLE_SIZE_QUERY = `SELECT c.relname AS name,
+pg_size_pretty(pg_total_relation_size(c.oid)) AS size
+FROM pg_class c
+LEFT JOIN pg_namespace n ON (n.oid = c.relnamespace)
+WHERE NOT n.nspname LIKE ANY($1)
+AND n.nspname !~ '^pg_toast'
+AND c.relkind='r'
```

Comment:
```suggestion
AND c.relkind = 'r'
```

Reply:
want me to remove these or do you want to @sweatybridge?

Reply:
I just committed them. Do you mind running through all the commands again to confirm everything works?